packages/quantic/cypress/page-objects/actions/action-perform-search.ts

```diff
@@ -0,0 +1,2 @@
+export const performSearch = () =>
```

Comment:
Could we put all "toolbar" actions in one file with a more general name or is it worth while to have this action in it's own `perform-search` file?

Reply:
I see actions as self-contained components so we can only add the ones we need when testing a quantic component. I think it is best to have one action file per component.